integration-tests/tests.ts

```diff
@@ -1277,20 +1283,19 @@ test("feature_flag_in_function", async ({ page }) => {
     await gotoAST(page);
     await awaitAnalysisLoad(testInfo);
 
-    // this await confirms that we have test_admin/stdlib/Test::one_v0 is in fact
+    // this await confirms that test_admin/stdlib/Test::one_v1 is in fact
     // in the autocomplete
     let before = Date.now();
     await page.type("#active-editor", "test_admin");
     await awaitAnalysis(page, before);
     await expectExactText(
       page,
       ".autocomplete-item.fluid-selected.valid",
-      "test_admin/stdlib/Test::one_v0Any",
+      "test_admin/stdlib/Test::one_v1Any",
```

Comment:
Question for @pbiggar: what's the "any" for here? how does this not break?

Reply:
The "Any" is the type. The text contents of the entire `div` is jammed together (even though they are in sibling sub-divs).